blog/2025/01/the-satellite-toolbox-ecosystem.md

```diff
@@ -0,0 +1,235 @@
+the packages to version 1.0 at the beginning of 2025. That way, we can officially declare
+the API stable.
+
+[SatelliteToolbox.jl](https://github.com/JuliaSpace/SatelliteToolbox.jl) v1.0 provides the
+following features to the Julia ecosystem:
+
+1. Convert between Keplerian elements and orbit state vectors.
+2. Convert between different types of anomalies when describing an orbit using Keplerian
+   elements.
+3. Obtain several space indices automatically from online sources.
+4. Compute the atmospheric density using multiple models (exponential, Jacchia-Roberts 1971,
+   Jacchia-Bowman 2008, and NRLMSISE-00). The required indices in each model can be
+   automatically fetched.
+5. Obtain the position vectors of the Sun and Moon, allowing us to perform several space
+   mission analysis tasks. For example, we need this information to check if our star
+   trackers will be blocked by those celestial bodies during the mission’s operational
+   lifetime.
+6. Compute the geomagnetic field using the IGRF model.
+7. Obtain the gravity field derivative, gravitational acceleration, and the gravity
+   acceleration using models obtained from [ICGEM](https://icgem.gfz-potsdam.de/home) files.
+   This information is crucial to develop numerical orbit propagators.
+8. Read, parse, and generate two-line elements (TLE).
+9. Fetch TLEs of any catalogued satellite from online sources.
+10. Propagate satellite orbits using several types of analytical algorithms (J2, J2
+    osculating, J4, J4 osculating, SGP4/SDP4, and two-body).
+11. Fit mean elements for any supported algorithms given a set of state vectors.
+12. Perform transformations between any reference system defined by the conventions
+    IAU-76/FK5 and IAU-2006/2010A.
+13. Perform transformations between geocentric and geodetic coordinates.
+14. Transform vectors from an Earth-Centered, Earth-Fixed reference frame to local reference
+    frames.
+15. Fetch and parse the Earth Orientation Parameters (EOP) from online sources, which can be
+    used to improve the accuracy of the coordinate transformations.
+16. Convert time between the required time epochs (UTC, UT1, TT).
+
+All those algorithms are written entirely in Julia and don’t need any external libraries.
+Hence, they run super fast! All the functions are also well-documented.
+
+Wow, what a journey it’s been! I want to give a big shoutout to everyone in the Julia
+community and contributors who helped make
+[SatelliteToolbox.jl](https://github.com/JuliaSpace/SatelliteToolbox.jl) what it is today.
+This tool has become a game-changer for our workflow at INPE, and it’s made Julia and its
+ecosystem a part of the Brazilian Space Program 🇧🇷!
+
+## The Future
+
+With our solid base and extensive testing, I will be implementing a range of analysis
+algorithms in the [SatelliteAnalysis.jl](https://github.com/JuliaSpace/SatelliteAnalysis.jl)
+package. This will empower Julia users to perform various space mission tasks effortlessly.
```

Comment:
```suggestion
package. This will empower Julia users to perform various space mission tasks.
```
Somehow I doubt this will be effortless. "More easily", perhaps. Though I think empower captures it well alone.